Sources/Commands/SwiftTestTool.swift

```diff
@@ -392,13 +394,31 @@ public struct SwiftTestTool: SwiftCommand {
             library: library
         )
 
+        // If the quiet flag is present, only output on failure.
+        let runQuietly = swiftTool.options.logging.quiet
+
+        // No need to close this stream.
+        // Buffered output's close method is a no-op.
+        var outputStream = BufferedOutputByteStream()
```

Comment:
I'd advise against using `BufferedOutputByteStream`. It's just a mutable string under the hood, with an added amount of confusing indirection, and (as the TSC library itself) will be deprecated in the future. Simplest alternative is plain `var stringAccumulator = ""` instead. You also won't need to convert to bytes and back to string again on L417-L421.

Reply:
Good to know. Thanks for the insight @MaxDesiatov